apps/mitt-konto/package.json

```diff
@@ -9,7 +9,9 @@
     "less": "^3.8.1",
     "react": ">=15.5",
     "react-date-picker": "^7.5.0",
-    "react-dom": ">=15.5"
+    "react-dom": ">=15.5",
+    "spago": "^0.15.2",
```

Comment:
We probably don't need `spago` or `yarn` here right? 🤔

Reply:
`yarn upgrade` or something must have automatically added them, I know I didn't do it myself.